baybe/simulation.py

```diff
@@ -607,9 +607,17 @@ def _look_up_target_values(
         #   column ordering, which is not robust. Instead, the callable should return
         #   a dataframe with properly labeled columns.
 
+        # For evaluating the lookup function, we need to filter the task parameter
```

Comment:
Hey @AVHopp, just saw this change. I might overlook something, but to me it seems this is a fix a the wrong place. According to the logic I mentioned last week, the lookup should very well process the task parameter, in my opinion. It's rather that the callable itself needs to be adjusted. So as far as I can tell, this is not a problem with the simulation functionality, but rather the input provided. Or what is your rationale?

Reply:
So just to make this clear: You'd then propose that instead of having multiple callables, one per task, as here: 
<img width="541" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/66a7d691-3072-4f01-ba33-30ca9f229dd3">
should be replaced by a single callable which basically uses the task parameter to determine which of the functions to ealuate?

Reply:
This would then break our logic of "a callable is something that accepts a list of arbitrary floats": We then need to allow either an explicit task parameter, string, or something similar.

Reply:
As discussed, I dropped the corresponding change.